lambda-loader/src/data.py

```diff
@@ -57,6 +59,28 @@ def create_support_table(self, table_name, key):
             BillingMode='PROVISIONED',
         )
 
+    def pasre_time(self, time_str):
+        # the -3 converts from nanoseconds to microseconds which the datetime class can handle
+        time = datetime.strptime(time_str[:-3], '%Y-%m-%d %H:%M:%S.%f')
+        return int(time.timestamp())
+
+    def put_time(self, region, time, region_name="Region",player_name="PlayerName", rank_name="Rank", rating_name="Ratings"):
+        item = {
+            player_name:timeToken,
+            region_name:region,
+            'Time': time,
+            rank_name: -1,
+            rating_name: [],
```

Comment:
```suggestion
            rating_name: [-1],
```
An empty rating list currently breaks a lot of the recap functions